stubs/pywin32/_win32typing.pyi

```diff
@@ -1175,20 +1175,24 @@ class PyMSG: ...
 class PyNETRESOURCE:
     @property
     def dwScope(self) -> int: ...
+    @dwScope.setter
+    def dwScope(self, value: int | bool) -> None: ...
     @property
     def dwType(self) -> int: ...
+    @dwType.setter
+    def dwType(self, value: int | bool) -> None: ...
     @property
     def dwDisplayType(self) -> int: ...
+    @dwDisplayType.setter
+    def dwDisplayType(self, value: int | bool) -> None: ...
```

Comment:
These unions are redundant, since `bool` is a subclass of `int` (_but_, I understand that this can be surprising, so I'm open to leaving this as it is if `bool` is treated differently to `int` in some way):

```suggestion
    def dwScope(self, value: int) -> None: ...
    @property
    def dwType(self) -> int: ...
    @dwType.setter
    def dwType(self, value: int) -> None: ...
    @property
    def dwDisplayType(self) -> int: ...
    @dwDisplayType.setter
    def dwDisplayType(self, value: int) -> None: ...
```

Reply:
TIL. I knew about `int`/`float` (ie: they're special cased by type checkers because of the [numeric tower]), but I never realized `bool` actually subclassed `int` 🤯 

```py
>>> bool.__bases__
(<class 'int'>,)
```

Reply:
Yup! In early versions of Python there was no `bool` class, only `int`; having `bool` subclass `int` meant it was fully backwards-compatible